docs/UserGuide.md

```diff
@@ -181,6 +144,92 @@ Expected output (fail):
 Oops! Invalid search expression, please check again.
 ```
 
+### Viewing a fosterer's detail : `view`
+
+Views a fosterer's details by shifting the foster family list view page to a fosterer's profile view page.
+
+Format: `view INDEX`
+
+Parameters:
+* `INDEX`: Index of a fosterer to view their details shown in the list obtained by the find/list command
+
+<div markdown="span" class="alert alert-primary">
+  :exclamation: <b>Important:</b>
+Some commands are not available while on profile view page. 
+The list of available commands are: 
+<ul>
+  <li> <code>help</code> </li>
+  <li> <code>edit</code> </li>
+  <li> <code>exit</code> command to <b>exit the profile view page</b> </li>
+</ul>
+</div> 
+
+Example:
+* `list` followed by `view 2` to view the profile of the 2nd fosterer in the address book.
+
+Expected output (success):
+```agsl
+Viewing the profile of a fosterer...
+```
+Expected output (fail):
+```agsl
+Oops! Invalid fosterer index provided, please check again.
```

Comment:
right now when I test the output i got is "The person index provided is invalid" though HAHAHA

Reply:
ACTUALLYYY mine doesnt start with OOps HAHHA but for the rest im not sure if they changed